CONTRIBUTING.md

```diff
@@ -14,12 +14,12 @@ Patches are welcomed as pull requests on
 [our Github](https://github.com/dusk-network/plonk), as well as by
 email (preferably sent to all of the authors listed in `Cargo.toml`).
 
-If you want to work on an issue, please let us know in the comments of it and
+If you want to work on an issue, please let us know in the comments it and
```

Comment:
I'm not sure if the original sentence was correct grammatically speaking, but I got the sense of it. Now I don't get the meaning... @moCello do you agree?